{"uuid":"29e87aa4-a7b3-488b-92a6-5165987d19d0","historyId":"82eba1ced9d303b87915959507050eaf","fullName":"com.docsService.Api.test.DocsService_API.document_meta_data_Docs_service_TC1","labels":[{"name":"package","value":"com.docsService.Api.test.DocsService_API"},{"name":"testClass","value":"com.docsService.Api.test.DocsService_API"},{"name":"testMethod","value":"document_meta_data_Docs_service_TC1"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.docsService.Api.test.DocsService_API"},{"name":"host","value":"BNG-3Z8DYZ2"},{"name":"thread","value":"19020@BNG-3Z8DYZ2.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"document_meta_data_Docs_service_TC1","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"C:\\Users\\NG2574\\API_workspace\\AutomationTest\\resorces\\API_DocsService_Automation.xlsx (The process cannot access the file because it is being used by another process)","trace":"java.io.FileNotFoundException: C:\\Users\\NG2574\\API_workspace\\AutomationTest\\resorces\\API_DocsService_Automation.xlsx (The process cannot access the file because it is being used by another process)\r\n\tat java.base/java.io.FileOutputStream.open0(Native Method)\r\n\tat java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)\r\n\tat com.Api.utils.FileLib.setCellData(FileLib.java:52)\r\n\tat com.docsService.Api.test.DocsService_API.document_meta_data_Docs_service_TC1(DocsService_API.java:44)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1663745716507,"stop":1663745718924}